### PR TITLE
Add materials for the Code Image Generator tutorial

### PR DESCRIPTION
**Pull request by acsany**
**Where to put new files:**

- New files should go into a top-level subfolder, named after the article slug. For example: `my-awesome-article`

**How to merge your changes:** 

1. [Make sure the CI code style tests all pass (+ run the automatic code formatter if necessary).](https://github.com/realpython/materials/blob/master/README.md)
2. Find an RP Team member on Slack and ask them to review & approve your PR.
3. Once the PR has one positive ("approved") review, GitHub lets you merge the PR.
4. 🎉

### CODE REVIEW

File: code-image-generator/source_code_step_01/templates/code_input.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{% extends "base.html" %}
+{% block title %}
+  - Code Input
+{% endblock title %}
+{% block content %}
+{% endblock content %}
```
Comment (lpozo) [inline code-image-generator/source_code_step_01/templates/code_input.html:6]: We can add the spaces to match the code in the article.

File: code-image-generator/source_code_step_02/app.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    Flask,
+    redirect,
+    render_template,
+    request,
+    session,
+    url_for,
+)
+
+app = Flask(__name__)
+app.secret_key = "mysecretkey"
```
Comment (lpozo) [inline code-image-generator/source_code_step_02/app.py:11]: We're not using this secret key any longer.

File: code-image-generator/source_code_step_02/static/styles.css [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+}
+
+.controls {
+    margin: 1em 0;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-wrap: wrap;
+
+}
```
Comment (lpozo) [inline code-image-generator/source_code_step_02/static/styles.css:47]:
```suggestion
    flex-wrap: wrap;
}
```

File: code-image-generator/source_code_step_03/app.py [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+    session,
+    url_for,
+)
+from pygments import highlight
+from pygments.formatters import HtmlFormatter
+from pygments.lexers import Python3Lexer
+from pygments.styles import get_all_styles
+
+app = Flask(__name__)
+app.secret_key = "mysecretkey"
```
Comment (lpozo) [inline code-image-generator/source_code_step_03/app.py:15]: secret_key

File: code-image-generator/source_code_step_03/static/styles.css [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+}
+
+.controls {
+    margin: 1em 0;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-wrap: wrap;
+
+}
```
Comment (lpozo) [inline code-image-generator/source_code_step_03/static/styles.css:47]:
```suggestion
    flex-wrap: wrap;
}
```

File: code-image-generator/source_code_step_03/templates/base.html [hunk 6]
```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1"/>
+    <title>
+      Code to Image {% block title %}{% endblock %}
+      Code to Image
```
Comment (lpozo) [inline code-image-generator/source_code_step_03/templates/base.html:8]:
```suggestion
      Code to Image {% block title %}{% endblock %}
```

**Review by lpozo — CHANGES_REQUESTED** [review]
@acsany Hey!

I flagged a few details in the steps. It'd be nice to fully check the code with the article.

I typically create the step folders once I finish the TR updates. At this point, it should be a matter of following the article.

**Review by KateFinegan — APPROVED** [review]

**Pull request merged**